client/my-sites/marketing/connections/connection.jsx

```diff
@@ -46,6 +46,12 @@ class SharingConnection extends Component {
 		defaultServiceIcon: {
 			google_my_business: 'institution',
 			slack: 'link',
+			'instagram-basic-display': 'user',
```

Comment:
Haha it bugs me that this is the only one in single quotes but it seems like that is how it should be.

Reply:
Yeah, it would look better without the quotes. :)

Thank you or the review!